### PR TITLE
adds new privacy policy

### LINKED ISSUE #116: Add privacy policy page & link from the footer

### PR DESCRIPTION
**Pull request by r-ferrier**
Fixes #116

## Description

- Adds a default english privacy policy page, text copied from https://www.wildlifetrusts.org/privacy-policy

@geeksforsocialchange/developers

### CODE REVIEW

File: content/pages/privacy-policy.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+- Continuously improve our website for you
+
+We do NOT use cookies to:
+
+- Collect any personally identifiable information (without your express permission)
+- Collect any sensitive information (without your express permission)
+- Pass personally identifiable data to third parties
+- Pay sales commissions
+
+You can learn more about all the cookies we use below.
```
Comment (katjam) [inline content/pages/privacy-policy.md:301]: We need to cut this list down to only the cookies we use (Us - to remember cookie settings, GA & Hotjar)

File: content/pages/privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+
+Our cookies help us:
+
+- Make our website work as you'd expect
+- Remember your settings during and between visits
+- Improve the speed/security of the site
+- Allow you to share pages with social networks like Facebook
+- Continuously improve our website for you
+
+#### c) Sensitive personal data
```
Comment (katjam) [inline content/pages/privacy-policy.md:162]: Probably need to alter this section too - we'll specifically collect in the case studies form. I have flagged in our cycle tasks that we need them to review this. So probably OK to leave as is here.

**Review by katjam — CHANGES_REQUESTED** [review]
Remove the specific cookies we will not use. I think the other sections need to be reviewed by them before we make changes. Especially as we'd need to keep in sync with the Welsh.

File: content/pages/privacy-policy.md (hunk 1 above)
Comment (r-ferrier) [reply content/pages/privacy-policy.md:301]: I've removed the cookie providers we don't use and made a new issue to audit the exact cookies we use at the end of development - #132 -  I've not added it to this milestone.

File: content/pages/privacy-policy.md (hunk 2 above)
Comment (r-ferrier) [reply content/pages/privacy-policy.md:162]: I've also made a new issue for this - #133 -  but let me know if that's overkill and I can just remove it!

**Review by katjam — APPROVED** [review]
Great!

**Comment by katjam** [comment]
@r-ferrier can I merge?

**Pull request merged**